client/app/upgrades/v_1_2/constants.go

```diff
@@ -0,0 +1,49 @@
+//nolint:revive,stylecheck // versioning
+package v_1_2
+
+import (
+	"time"
+
+	storetypes "cosmossdk.io/store/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/piplabs/story/client/app/keepers"
+	"github.com/piplabs/story/client/app/upgrades"
+)
+
+const (
+	UpgradeName = "v1.2"
+
+	// AeneidUpgradeHeight defines the block height at which the upgrade is triggered on Aeneid.
+	AeneidUpgradeHeight = 10_000_000
+	// StoryUpgradeHeight defines the block height at which the upgrade is triggered on Story.
+	StoryUpgradeHeight = 10_000_000
+
+	// Initial parameters.
+	InitialRefundFeeBps = 100                // 1%
+	InitialRefundPeriod = 7 * 24 * time.Hour // 7 days
```

Comment:
qq. We want to set 7 days for initial refund period, not 1 day?

Reply:
right, I recall we talked about the parameter value. Changed to 1 day here d2404ff, please confirm that 1 day is the intended duration @gezerui0124 @edisonz0718

Reply:
And please check the refundFeeBps as well 🙏